src/index.html

```diff
@@ -5,10 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <div class="menu__container">
+        <a href="#home" class="menu__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+
+        <nav class="nav">
+
+          <ul class="nav__list">
+
+            <li class="nav__item">
+              <a href="#apple" class="is-active nav__link">apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#laptops&computers"
```

Comment:
Please re-read this paragraph in checklist
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/097ea283-892c-4007-af67-f1da44ed337f)

Reply:
I've made changes to correct ones. Please check them.
And about the font-weight - pretty sure it was 500, but I cannot open figma to check what was it